babel.config.js

```diff
@@ -3,28 +3,23 @@
 module.exports = function (api) {
   const env = process.env.NODE_ENV;
   const babelEnv = process.env.BABEL_ENV;
-
+  const supportedBrowsers = require("@rei/browsers-config");
```

Comment:
nice!

Reply:
Figured I might as well take care of it while I'm in there